documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -47,6 +52,12 @@ The `routes` option allows you to customise the [`_routes.json`](https://develop
 
 You can have up to 100 `include` and `exclude` rules combined. Generally you can omit the `routes` options, but if (for example) your `<prerendered>` paths exceed that limit, you may find it helpful to manually create an `exclude` list that includes `'/articles/*'` instead of the auto-generated `['/articles/foo', '/articles/bar', '/articles/baz', ...]`.
 
+### `external`
+
+`external` is equivalent to the respective [_external_ option of esbuild](https://esbuild.github.io/api/#external).
+You can use it to mark a file or package as external to exclude it from your build.
+Typically, this can be used for packages that do import _NodeJS_ modules such as `fs`.
```

Comment:
we keep paragraphs together as a single line and let the editor take care of wrapping the lines
```suggestion
`external` is equivalent to the respective [`external` option of esbuild](https://esbuild.github.io/api/#external). You can use it to mark a file or package as external to exclude it from your build. For example, this can be used to exclude Node.js modules such as `fs`, which are not available on Cloudflare.
```

Reply:
In my experience, it makes diffs harder to read, especially when in long paragraphs only single things are changed, like a missing or extra letter. That's why I got used to splitting sentences up in Markdown files. It doesn't matter for the output anyway, since Markdown collapses single `\n` into a space.

I've adjusted to your wishes regardless.